lib/pbench/test/unit/server/test_user_auth.py

```diff
@@ -2,12 +2,45 @@
 from http import HTTPStatus
 import time
 
+import pytest
+
+from pbench.server.api.resources.users_api import AuthenticationSchema
 from pbench.server.database.database import Database
 from pbench.server.database.models.active_tokens import ActiveTokens
 from pbench.server.database.models.users import User
 from pbench.test.unit.server.conftest import admin_username, login_user, register_user
 
 
+class TestAuthenticationSchema:
+    @pytest.mark.parametrize("schema", ("dev", "prod", "test", "", "bad_schema"))
+    def test_schema(self, client, server_config, schema):
+        if schema == "":
+            schema = "prod"
```

Comment:
This seems pointless: if you're going to change `""` to `"prod"` why even include `""` in the list?